### DISCUSSION TITLE
How can i change the css style of final generaged ui?

### DISCUSSION THREAD

**Discussion by quchunguang** [zauberzeug/nicegui#1572] [Q&A]
### Question

For example, `example/slideshow` shows the slides with some padding space on `div#c3`. How can i get rid of it? 
I tried,
```py
slide = ui.image(f'slides/{files[index]}').classes('p-0')
```
but not work.

**Comment by rodja** [comment]
The image does not have a margin, but the div with id `c3` and class `nicegui-content` has a padding. To reach into the DOM and change it you can use:

```py
ui.query('.nicegui-content').classes('p-0')
```

I've updated the example to include this line. When the code was first written there was not `ui.query` call... but I think its great to include it in this example.
